### ISSUE TITLE
Redundancy in the "patronizing" section

### ISSUE THREAD

**Issue by dbooth-boston** [w3c/PWETF#265]
1. There is a lot of redundancy between these two bullets in the "patronizing" section.  They are both about making assumptions about people's skills, though they use different examples:

             <li>Intentionally or unintentionally making assumptions about the skills or knowledge of others, such as using language that implies the audience is uninformed on a topic (e.g. interjections like "I can't believe you don't know about [topic]").
             </li>

             <li>Assuming that particular groups of people are technically unskilled due to their characteristics (e.g., “So easy your grandmother could do it”, which implies an older woman might not be technically competent).
             </li>

I think the redundancy should be reduced or eliminated.  This is the main editorial issue that I see in these bullets.

2. A secondary editorial issue is that the example in the first bullet above ("I can't believe you don't know about [topic]") implies that the audience is woefully uninformed, which is patronizing in itself, regardless of whether the speaker made inappropriate assumptions about the audience's knowledge.  So the preface about making assumptions about the audience's knowledge seems unnecessary in that bullet.

3. A third and minor editorial issue is that I think the wording of these two bullets could be more simplified in other small ways, which I'll illustrate below.

4. Incorporating the above, I suggest simplifying these two bullet examples of patronizing language or behavior to something like the following:

             <li>Implying that the audience is unusually uninformed (e.g. making statements like "I can't believe you don't know about [topic]").
             </li>

             <li>Implying that certain demographic groups are unskilled (e.g., “So easy your grandmother could do it”, which implies an older woman would not be technically competent).
             </li>

**Comment by dbooth-boston** [comment]
Based on discussion in today's meeting, how about this wording instead, for the above two bullets?

> Intentionally or unintentionally implying that the audience is woefully uninformed (e.g. making statements like "I can't believe you don't know about [topic]").

> Assuming that certain groups are unskilled (e.g., “So easy your grandmother could do it”, which implies an older woman would not be technically competent).

The reason I included the word "woefully" above is because it is normal for an audience to be somewhat uniformed: the purpose of any presentation is to inform the audience of something they didn't already know.  

Potential variations that could be used instead of "woefully uninformed" include: "very uninformed", or "not adequately informed".

**Comment by TzviyaSiegman** [comment]
I think that "woefully" weakens this. The "audience" here is not just the audience in a setting like a conference talk. It could be a more informal audience, like a group of people in a working group. I strongly feel that modifying uniformed will weaken this. I don't think that these changes are significantly different than what we have.

**Comment by dbooth-boston** [comment]
I'm okay with dropping "woefully".

**Comment by dbooth-boston** [comment]
The following suggested rewording incorporates feedback given in the [23-May-2023 meeting](https://www.w3.org/2023/05/23-pwe-minutes.html#t08), and the comment above from @TzviyaSiegman :

> Intentionally or unintentionally implying that the audience is uninformed (e.g. making statements like "I can't believe you don't know about [topic]").

>  Assuming that certain groups are unskilled (e.g., “So easy your grandmother could do it”, which implies an older woman would not be technically competent).
